WireAPI/Sources/WireAPI/Models/UserClient/SimplifiedUserClient.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+public struct SimplifiedUserClient: Equatable, Codable, Sendable {
```

Comment:
I think this is the user client for other users, and the other one with all the metadata is only for self clients. Perhaps we can make this distinction more obvious with `SelfUserClient` and `OtherUserClient`.

Reply:
yes I aligned with Android for the naming but your suggestion I think gives more clarity, done in https://github.com/wireapp/wire-ios/pull/2019/commits/b9f535f64fb834f6a0409f69830e48d7c82c8f45